src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -6,74 +6,77 @@
         </div>
         <div class="widget-footer">
             <div class="footer-left">
-                <p-datetime-picker style-type="text"
-                                   select-mode="range"
-                                   :selected-dates.sync="proxySelectedDates"
-                />
-                <p-divider :vertical="true" />
-                <currency-select-dropdown :print-mode="printMode"
-                                          @update="handleUpdateCurrency"
+                <label v-if="dateLabel"
+                       class="widget-footer-label"
+                >{{ dateLabel }}</label>
+                <p-divider v-if="isDivided"
+                           :vertical="true"
                 />
+                <label class="widget-footer-label">{{ currencyLabel }}</label>
             </div>
             <div class="footer-right">
                 <slot name="footer-right">
-                    <router-link v-if="widgetLink && !printMode"
-                                 :to="widgetLink"
-                                 class="anchor-button"
+                    <component :is="fullDataTag"
+                               v-if="(widgetLink || widgetRoute) && !printMode"
+                               :href="widgetLink"
+                               :to="widgetRoute"
+                               class="anchor-button"
                     >
                         {{ $t('BILLING.COST_MANAGEMENT.DASHBOARD.FULL_DATA') }}
                         <p-i name="ic_arrow_right"
                              width="1rem"
                              height="1rem"
                              color="inherit transparent"
                         />
-                    </router-link>
+                    </component>
                 </slot>
             </div>
         </div>
     </div>
 </template>
 
 <script lang="ts">
-import type { PropType, SetupContext } from 'vue';
+import type { PropType } from 'vue';
 import {
     reactive, toRefs, defineComponent, computed,
 } from 'vue';
+import type { Route } from 'vue-router';
 
-import { PDatetimePicker, PDivider, PI } from '@spaceone/design-system';
+import { PDivider, PI } from '@spaceone/design-system';
+import dayjs from 'dayjs';
 
 import type { Currency } from '@/store/modules/display/config';
-import { CURRENCY } from '@/store/modules/display/config';
+import { CURRENCY_SYMBOL } from '@/store/modules/display/config';
 
-import { useProxyValue } from '@/common/composables/proxy-state';
+import type { WidgetOptions, WidgetSize } from '@/services/dashboards/widgets/config';
+import { WIDGET_SIZE } from '@/services/dashboards/widgets/config';
 
-import { WIDGET_SIZE } from '../config';
-import type { WidgetSize } from '../type';
 import CurrencySelectDropdown from './CurrencySelectDropdown.vue';
 
 interface Props {
     title: string;
     size: WidgetSize;
     width: string;
-    widgetLink: string;
+    widgetLink?: string;
+    widgetRoute: Route;
```

Comment:
```suggestion
    widgetRoute?: Route;
```
Looking at the definition of props below, the default value is `undefined`, so why not make this property optional?

Reply:
Also `dateRange`.